docs/Ramm.md

```diff
@@ -0,0 +1,186 @@
+
+# Ramm Contract Developer Documentation
+
+## Overview
+
+The `Ramm` contract is designed to manage liquidity and facilitate swaps between NXM tokens and ETH. It provides a dynamic pricing mechanism through ratcheting and circuit breakers to maintain stability. The contract includes features for managing ETH and NXM reserves, along with ensuring that swaps operate within predefined parameters and limits.
```

Comment:
The `Ramm` contract is designed to allow swaps between NXM tokens and ETH. Internally it works by simulating 2 uniswap v2 -like pools which have their liquidity adjusted using liquidity injection and a ratcheting mechanism.